### DISCUSSION TITLE
KServe with KubeFlow

### DISCUSSION THREAD

**Discussion by yurkoff-mv** [kubeflow/kfserving#2218] [General]
How can I simultaneously use the advantages of kubeflow in the form of Jupiter notebooks and pipelines and the advantages of kserve in the form of InferenceService?
How should I install these components into the cluster?

**Comment by flopach** [comment]
+1 exploring this as well

**Comment by markwinter** [comment]
There is a KServe component now for Kubeflow Pipelines
https://github.com/kubeflow/pipelines/tree/master/components/kserve

I think Kubeflow plans to stay with KFServing for the next year? So if you want to use Kubeflow + KServe, you should migrate the KFServing installation following this guide https://kserve.github.io/website/0.8/admin/migration/

**Comment by jagadeeshi2i** [comment]
You can try out this sample notebook. https://github.com/kubeflow/pipelines/blob/master/samples/contrib/pytorch-samples/Pipeline-Bert.ipynb. It uses Kubeflow pipelines for model building and KServe for model serving.

Also for installation of kubeflow along with KServe refer - https://github.com/kubeflow/manifests
